src/store/itemShare.ts

```diff
@@ -21,23 +23,25 @@ export const useShareStore = defineStore("share", {
   }),
   actions: {
     async connect(podcastname: string) {
+      this.inbox = {}
       // TODO: swap this out with a collectionGroup query
       usePodcastStore().getPodcasts.forEach((podcast: podcasts) => {
-        onSnapshot(
-          doc(db, podcastname, "inbox", podcast.id, "shares"),
-          (doc) => {
-            this.inbox[podcast.id] = doc.data()?.items ?? []
-          },
-        )
+        getDoc(
```

Comment:
getDoc i don't think does a subscribe to updates does it?

Reply:
'getDoc' is got only shared podcast items.
when we use 'onSnapshot' then all-time inbox data are updated then do not show properly inbox items.

Reply:
right, but the inbox needs to be constantly updated in realtime

Reply:
When an item is shared then I have re-called the 'getDoc' function so it automatically gets real-time inbox data.

Reply:
that is not real-time.
`user1` shares `item1` from `podcast1` to `podcast2`
`user2` looking at `podcast2` should see the `item1` appear without having to carry out any actions